Dockerfile

```
@@ -0,0 +1,42 @@
+RUN npm prune --production
+
+# ---
+FROM node:22-alpine AS production
+
+WORKDIR /app
+
+# Set NODE_ENV to production
+ENV NODE_ENV=production
+
+# Install wget for health checks
+RUN apk add --no-cache wget
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)